src/templates/service.yaml

```diff
@@ -1,10 +1,20 @@
 apiVersion: v1
 kind: Service
 metadata:
+  labels:
+    app: seldon
+    app.kubernetes.io/instance: 'seldon-core'
+    app.kubernetes.io/name: 'seldon-core-operator'
+    app.kubernetes.io/version: '1.6.0'
   name: seldon-webhook-service
+  namespace: {{ namespace }}
 spec:
   ports:
   - port: {{ webhook_port }}
     targetPort: {{ webhook_port }}
   selector:
     juju-app: {{ service }}
+    app.kubernetes.io/instance: seldon1
+    app.kubernetes.io/name: seldon
+    app.kubernetes.io/version: v0.5
```

Comment:
Is this value supposed to match the `1.6.0` value above?

Reply:
Seems like it is that way for both [Kubeflow](https://github.com/kubeflow/manifests/blob/master/contrib/seldon/seldon-core-operator/base/resources.yaml#L4830) and [Seldon](https://github.com/SeldonIO/seldon-core/blob/master/helm-charts/seldon-core-operator/templates/service_seldon-webhook-service.yaml) manifests.

Reply:
I've removed almost the whole block, except for `app.kubernetes.io/name: seldon`. Let me know if that is better.